kpi/utils/query_parser/query_parser.py

```diff
@@ -108,9 +125,23 @@ def notexp(text, a, b, elements):
 
     @classmethod
     def term(cls, text, a, b, elements):
-        if elements[0].text == '':
+
+        def _get_value(field, elements):
             # A search term by itself without a specified field
-            field = SEARCH_DEFAULT_FIELD_LOOKUP
+            value = elements[1]
```

Comment:
Personal preference. I would use a `_` suffix to `value` (and `elements`) to avoid any confusions with `value` (or `elements`) on line 136 & 153. (even it is in own scope and should not shadow `value` from outer scope)